src/Apps/Order2/Routes/Checkout/Hooks/useStripePaymentBySetupIntentId.tsx

```diff
@@ -43,11 +47,16 @@ export function useStripePaymentBySetupIntentId(
 
     // Convert string to boolean - URLSearchParams returns string "false" or "true"
     const shouldSaveBankAccount = save_bank_account === "true"
-    let oneTimeUse = !shouldSaveBankAccount
+    const oneTimeUse = !shouldSaveBankAccount
```

Comment:
Don't know how this and other edits in the file made it past the linter into main.